packages/geant4/package.py

```diff
@@ -27,14 +38,10 @@ def install(self, spec, prefix):
         # http://geant4.web.cern.ch/geant4/UserDocumentation/UsersGuides/InstallationGuide/html/ch02s03.html
         # fixme: likely things that need addressing:
         # -DGEANT4_USE_OPENGL_X11=ON
-        # -DGEANT4_USE_SYSTEM_CLHEP
-        # -DGEANT4_USE_SYSTEM_EXPAT
 
         if '+qt' in spec:
```

Comment:
Just a matter of taste. I'd favour the more explicit `if spec.variants['qt'] == true:`